src/test/java/net/openhft/chronicle/wire/Base85LongConverterTest.java

```diff
@@ -61,6 +61,25 @@ public void parse() {
         }
     }
 
+    @Test
```

Comment:
Please could you enhance the names of the test methods from `parseX` - I am not sure what some of them are testing